mautic/mautic
# Salesforce campaign segments

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | pending
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR puts Salesforce campaign members into a segment. First it syncs campaign members into  mautic, and resulting campaign members are pulled as Mautic contacts, these mautic contacts are then inputted in a segment, needs https://github.com/mautic/mautic/pull/3674

#### Steps to test this PR:
1. Apply this PR
1.5 Salesforce plugin must be configured, and fields matched
2. Create a segment with an Integration Campaign filter
3. Select the campaign you want to use as your segment filter
4. Update you segment
5. Check that Contacts in Mautic are put into this segment, and they coincide with campaign members in Salesforce
